src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/DownloadFileMittentePagoPATest.java

```diff
@@ -152,6 +152,190 @@ public void downloadECheckFile() {
 
     }
 
+    @And("Nella sezione Dettaglio Notifiche si scarica il documento allegato")
+    public void downloadDocumentiAllegati(){
+        logger.info("Si scaricano solo i documenti Allegati all'interno della notifica");
+
+        DettaglioNotificaMittenteSection dettaglioNotificaMittenteSection = new DettaglioNotificaMittenteSection(this.driver);
+        DettaglioNotificaSection dettaglioNotificaSection = new DettaglioNotificaSection(this.driver);
+        DataPopulation dataPopulation = new DataPopulation();
+
+        String workingDirectory = System.getProperty("user.dir");
+        String variabileAmbiente = System.getProperty("environment");
+        File pathCartella = new File(workingDirectory + "/src/test/resources/dataPopulation/downloadFileNotifica/mittente");
+        DownloadFile downloadFile = new DownloadFile(this.driver);
+        boolean headless = System.getProperty("headless").equalsIgnoreCase("true");
+        if (!downloadFile.controlloEsistenzaCartella(pathCartella)) {
+            pathCartella.mkdirs();
+        }
+        this.datiNotifica = dataPopulation.readDataPopulation("datiNotifica.yaml");
+        int count = 1;
+
+        dettaglioNotificaMittenteSection.waitLoadingSpinner();
+        String codiceIUN;
+        String destinatario = dettaglioNotificaMittenteSection.getInfoNotifica(1);
```

Comment:
try to separate this logic in other method

Reply:
this logic  is already separated from other method, further separation will complicate the method

Reply:
ok